src/api/lib/tasks/databases.rake

```diff
@@ -83,6 +83,20 @@ namespace :db do
       end
       File.open("#{Rails.root}/db/structure.sql", "w+") { |f| f << new_structure }
     end
+
+    desc "Verify that structure.sql in git is up to date"
+    task verify: :environment do
+      puts "Running rails db:migrate"
+      Rake::Task["db:migrate"].invoke
```

Comment:
Should this make sure that the database is empty before running? Otherwise if the database was loaded from structure.sql then this task would be pointless:

```ruby
Rake::Task["db:drop db:create db:migrate"].invoke
```